run_aggregate_loanbooks.R

```diff
@@ -138,17 +146,26 @@ matched_prioritized <- readr::read_csv(
 )
 
 # optional: apply sector split----
-if (apply_sector_split) {
-  companies_sector_split <- readr::read_csv(
-    file.path(input_path_matched, "companies_sector_split.csv"),
-    col_types = col_types_companies_sector_split,
-    col_select = dplyr::all_of(col_select_companies_sector_split)
-  )
+if (apply_sector_split & sector_split_type_select %in% c("equal_weights", "worst_case")) {
```

Comment:
NIT: 
`sector_split_type_select %in% c("equal_weights", "worst_case")` here is superfluous in `apply_sector_split & sector_split_type_select %in% c("equal_weights", "worst_case")` 

The latter `if` statements already assert that `sector_split_type_select` must be in one of `equal_weights` or `worst_case`

Reply:
removing it could in theory lead to entering the control flow with an invalid `sector_split_type_select`. In that case the script would attempt to run `apply_sector_split_to_loans()` without loading any sector split file. So if anything I think it should remain in the if condition and I should make the inner `else if` a simple `else`

Reply:
Ah apologies, I misunderstood where the closing `}` was. Right you are!

Reply:
simplified